frontend/src/CodalabApp.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { Router, Route, Link, Redirect, withRouter, Switch } from 'react-router-dom';
+import { CookiesProvider, withCookies } from 'react-cookie';
+
+import { MuiThemeProvider } from '@material-ui/core/styles';
+import CodalabTheme from './theme';
+
+// Components
+import UserInfo from './components/UserInfo';
+import HomePage from './components/HomePage';
+import $ from 'jquery';
+import NavBar from './components/NavBar';
+import Footer from './components/Footer';
+import Login from './components/Login';
+import { SignUp, SignUpSuccess } from './components/SignUp';
+import { ChangeEmail, ChangeEmailSuccess } from './components/ChangeEmail';
+import VerifySuccess from './components/VerifySuccess';
+import VerifyError from './components/VerifyError';
+import Worksheet from './components/worksheets/Worksheet';
+import {
+    PasswordReset,
+    PasswordResetSent,
+    PasswordResetVerified,
+    PasswordResetComplete,
+} from './components/PasswordReset';
+
+// Routes
+import BundleRoute from './routes/BundleRoute';
+
+import history from './history';
+import Cookies from 'universal-cookie';
+
+////////////////////////////////////////////////////////////
+// 1. Click the public page
```

Comment:
What's this?

Reply:
random stuff left over from tutorials :)